src/renderers/WebGLRenderer.js

```diff
@@ -1414,6 +1439,8 @@ function WebGLRenderer( parameters ) {
 		object.modelViewMatrix.multiplyMatrices( camera.matrixWorldInverse, object.matrixWorld );
 		object.normalMatrix.getNormalMatrix( object.modelViewMatrix );
 
+		object.interpolatedDiffuseLightProbe = computeInterpolatedLightProbe( object, scene );
```

Comment:
Maybe this should be `interpolatedDiffuseSH` rather than a LightProbe instance? 🤔

Once we're actually doing interpolation between probes, we won't need a LightProbe to represent the interpolated result of other probes. For example:

```js
// Update .interpolatedDiffuseSH (instance of SphericalHarmonics3)
computeInterpolatedSH( object, scene, object.interpolatedDiffuseSH );
```